components/member-card/show-wallet.js

```diff
@@ -0,0 +1,76 @@
+import classNames from 'components/member-card/card.module.scss';
+import { CURRENCIES, walletURL } from 'helper-functions/urls';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+const Coins = (props) => {
+  const { coin, balance } = props;
+  return (
+    <div className={classNames.walletContainer}>
+      <div className={classNames.coinData}>
+        <p>{coin}</p>
+        <p>{balance}</p>
+      </div>
+      <div
+        className={
+          coin === CURRENCIES.NEELAM ? classNames.coinNeelam : classNames.coinDinero
+        }></div>
+    </div>
+  );
+};
+
+const ShowWallet = ({ show }) => {
+  const [currencies, setCurrencies] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const coins = [];
+  useEffect(() => {
+    const fetchData = async () => {
+      await fetch(walletURL, { credentials: 'include' })
+        .then((res) => {
+          if (res.status >= 400 && res.status < 600) {
+            throw new Error('Bad response from server');
+          }
+          return res.json();
+        })
+        .then((resJson) => {
```

Comment:
Why mix of async-await and .then?
Let's stick to async await please

Reply:
Okay I'll change it